### ISSUE TITLE
[FileSelect] Reselection throws an exception when multiple option is disabled

### ISSUE THREAD

**Issue by maca88** [telerik/kendo-angular#2737]
**Describe the bug**
Reselecting a file on a `FileSelect` component throws an exception when using `[multiple]="false"`.

**To Reproduce**
1. Open: https://stackblitz.com/edit/angular-sdeb5z
2. Click on `Select files..` button and select a file
3. Open the console to see the exception
```
ERROR TypeError: Cannot read property 'map' of null
    at eval (main.js:5)
    at SafeSubscriber.schedulerFn [as _next] (event_emitter.ts:117)
    at SafeSubscriber.__tryOrUnsub (Subscriber.ts:265)
    at SafeSubscriber.next (Subscriber.ts:207)
    at Subscriber._next (Subscriber.ts:139)
    at Subscriber.next (Subscriber.ts:99)
    at EventEmitter.Subject.next (Subject.ts:70)
    at EventEmitter.emit (event_emitter.ts:86)
    at e.onChange (main.js:5)
    at e.clearFiles (main.js:5)
```

**Expected behavior**
No exception should be thrown and the `ClearEvent` event should printed in the console.

**Browser**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 79

**Comment by Raisolution** [comment]
Fixed in the latest dev version of the `kendo-angular-upload` package.

**Comment by Raisolution** [comment]
Fix is available in `v5.2.1`
